hub/util/remove_cache.py

```diff
@@ -9,3 +9,9 @@ def remove_memory_cache(storage: StorageProvider):
     ):
         return storage.next_storage
     return storage
+
+
+def remove_all_cache(storage: StorageProvider):
+    while isinstance(storage, LRUCache):
+        storage = storage.next_storage
+    return storage
```

Comment:
docstring? also, can you rename this method?

Reply:
I'll add a doc string but why rename? it's literally removing all the caches and getting underlying storage.